tox.ini

```diff
@@ -16,7 +16,13 @@ commands =
     stestr run {posargs}
 whitelist_externals =
```

Comment:
```suggestion
allowlist_externals =
```
Looks like tox also removed support of whitelist_externals keyword, thus breaking the build.

Reply:
I should also add that whitelist_externals was removed in December in version 4. In which case, we should also set minversion in the tox.ini as 4.

https://tox.wiki/en/latest/changelog.html#v4-0-0rc4-2022-12-06